Sources/MapboxMaps/Gestures/GestureHandlers/DoubleTapToZoomGestureHandler.swift

```diff
@@ -30,10 +30,14 @@ internal class DoubleTapToZoomGestureHandler: GestureHandler {
         case .ended:
             cameraAnimationsManager.cancelAnimations()
             delegate?.gestureBegan(for: gestureType)
+            delegate?.gestureEnded(for: self.gestureType, willDecelerate: true)
             _ = cameraAnimationsManager.ease(to: CameraOptions(zoom: mapboxMap.cameraState.zoom + zoomDelta),
                                              duration: 0.3,
                                              curve: .easeOut,
-                                             completion: nil)
+                                             completion: { [weak self] _ in
```

Comment:
I believe @macdrevx has a PR that makes ease a discardable result. So maybe land that PR first and then you should rebase